interfaces/automated-testing/scd/scd.yaml

```diff
@@ -557,3 +600,32 @@ paths:
       operationId: deleteFlight
       description: >-
         This endpoint simulates the operator intention to cancel or end a flight operation.
+
+  /v1/areas:
```

Comment:
You're right that my endpoint wasn't very RESTful, but I'm hesitant to use verbs because [they're weird](https://google.aip.dev/136) -- I've instead restructured the endpoint to be more resource-based.  The new logical resource is "a request to clear an area" and the client POSTs to that resource collection to create a new resource of that type.  Hypothetically, we could add more supporting methods (like a GET for a particular request) which would make asynchronous implementation easy, but that seems unnecessary at this point.